go/pkg/tool/tool_test.go

```diff
@@ -215,7 +215,7 @@ func TestTool_UploadBlob(t *testing.T) {
 	cas := e.Server.CAS
 
 	tmpFile := path.Join(t.TempDir(), "blob")
-	if err := ioutil.WriteFile(tmpFile, []byte("Hello, World!"), 777); err != nil {
+	if err := ioutil.WriteFile(tmpFile, []byte("Hello, World!"), 0777); err != nil {
```

Comment:
It is better to have separate PR for this?
Or this is necessary for this PR?

Reply:
https://github.com/bazelbuild/remote-apis-sdks/pull/219